src/components/prescriptions/PrescriptionImportComponent..vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <li class="documentBox">
+        <font-awesome-icon :icon="classOfIcon" />
+        <h5 class="documentBoxtitle">{{ name }}</h5>
+        <h5 class="documentBoxtitle">{{ date }}</h5>
```

Comment:
same problem. It is no semantic to add to h5  when we have relation: date depends to name